src/test/resources/inputs/basics/exclude.sql

```diff
@@ -212,7 +212,7 @@ SELECT * EXCLUDE t.flds.a, t.flds.b, t.flds.c FROM EXCLUDE_T AS t;
 
 --#[exclude-49]
 -- Exclude two nested fields; same transpiled query (other than table name) as #[exclude-04]
-SELECT * EXCLUDE t.flds.b, t.flds.c.field_x FROM EXCLUDE_T_NULLABLE AS t;
```

Comment:
self-review: in PLK-1.x, all types are nullable so the table `EXCLUDE_T_NULLABLE` is the same as `EXCLUDE_T`. changed to `EXCLUDE_T` in the inputs and outputs.